config/deps/feature_flags.exs

```diff
@@ -1,12 +1,7 @@
 import Config
 
 config :laboratory,
```

Comment:
Can we remove laboratory now or is it still being used for something else?

Reply:
Oh nice catch, it's just this one here. I can remove